apps/server/src/server.ts

```diff
@@ -16,6 +16,7 @@ const lyricProvider = provider(defaultAppConfig);
 const app = express();
 
 app.use(helmet());
+app.use(json());
```

Comment:
Express middleware function to parse json requests

Reply:
We may want to add an optional limit for those.
In this example I'm giving a fixed 50mb, but this must be something passed as an env var instead.

```suggestion
app.use(json({ limit: '50mb' }));
```

Reply:
`limit` property added on the `SubmittedDataRouter` file. as this is the only router parsing json.

Reply:
I think the json middleware has a pretty sane default that I've only ever had to change in order to allow LARGER files.

Reply:
yes, indeed just added limit property to `Multer` middleware, which by default `Multer` has unlimited file size